web/themes/custom/badcamp2017/sass/components/_featured.scss

```diff
@@ -1,16 +1,18 @@
-.section-featured {
-  background-color: transparent;
-  .section--inner {
-    background-image: url("../images/homepage-bg-image.jpg");
+
+.path-frontpage {
+  #zone-highlighted {
```

Comment:
this is nitpicky, but it's generally a best practice to use classes instead of IDs, as IDs are reserved for JS use.